docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md

```diff
@@ -2,22 +2,14 @@
 title: Event name cleanup in Subscriptions
 ---
 
-To conform to Cloud Event specifications, sometimes Eventing must modify the event names to filter out non-alphanumeric character. In this tutorial, we will show one such example of event name cleanup.
-We will create a [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) having alphanumeric characters in the event names and see how Eventing behaves. You can read more about Event name format and cleanup [here](../../05-technical-reference/evnt-01-event-names.md).
+To conform to Cloud Event specifications, sometimes Eventing must modify the event names to filter out non-alphanumeric characters. This tutorial presents one example of event name cleanup.
+You learn how Eventing behaves when you create a [Subscription](../../05-technical-reference/00-custom-resources/evnt-01-subscription.md) having alphanumeric characters in the event names. Read more about [Event name format and cleanup](../../05-technical-reference/evnt-01-event-names.md).
 
 ## Prerequisites
 
-1. Provision a [Kyma Cluster](../../02-get-started/01-quick-install.md).
-2. (Optional) Deploy [Kyma Dashboard](../../01-overview/main-areas/ui/ui-01-gui.md) on the Kyma cluster using the following command. Alternatively, you can also use `kubectl` CLI.
-   ```bash
-   kyma dashboard
-   ```
-3. (Optional) Install [CloudEvents Conformance Tool](https://github.com/cloudevents/conformance) for publishing events. Alternatively, you can also use `curl` to publish events.
-   ```bash
-   go install github.com/cloudevents/conformance/cmd/cloudevents@latest
-   ```
+Follow the [prerequisites steps](../../02-get-started/04-trigger-workload-with-event.md#prerequisites) mentioned in the getting-started guide.
 
-## Create a Workload
+## Create a Function
 
```

Comment:
The difference between the following instructions and the ones in `../../02-get-started/04-trigger-workload-with-event.md#create-a-function` is only the sample code, right?

If yes, then you could replace this whole section with something like:

```suggestion
2. Create a Function by following the [instructions](../../02-get-started/04-trigger-workload-with-event.md#create-a-function) in the Getting Started guide.

> **NOTE:** For this example, instead of the default code sample, replace the Function source with the following code:
```

Reply:
updated